src/interp.rs

```diff
@@ -261,12 +299,46 @@ impl<'a, 'tcx> Machine<'a, 'tcx> {
     /// location inside the Machine.
     fn eval_place(&mut self, place: &Place<'tcx>) -> Address {
         match *place {
-            Place::Local(local) => Address::Local(local),
+            Place::Local(local) => Address::Local(local, 0),
             Place::Static(ref static_) => unimplemented!(),
-            Place::Projection(ref proj) => unimplemented!(),
+            Place::Projection(ref proj) => {
```

Comment:
So what's a "projection"?

Reply:
A projection is a field access of index into an aggregate type. It can be used in other cases, like slices and reference chasing, but this is the most common use case.